### PR TITLE
OCRAD-23 :- add concept reference term type document.

### PR DESCRIPTION
**Pull request by Akayeshmantha**

### CODE REVIEW

File: Concepts/concept_reference_term_type.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Concepts Reference Term Type
```
Comment (bmamlin) [inline Concepts/concept_reference_term_type.md]: `# Concept Reference Term`

Singular "Concept" and remove the word "Type"

File: Concepts/concept_reference_term_type.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+* A concept term defines a medical coding term or OpenMRS concept dictionary term that could be mapped to a concept. 
+A medical coding term is used to represent a name from a standard medical code (ie.  SNOMED, ICD10, LOINC, etc). OpenMRS 
+concept dictionary terms are used to refer to concepts by concept_id or name of a concept in the concept dictionary (ie. CIEL, PIH, AMPATH, etc.).  
+  
+## Available operations. 
+1.  [List concept reference term types](#list-concept-reference-term-types)
+2.  [Create a concept reference term type](#create-a-concept-reference-term-type)
+3.  [Update a concept reference term type](#update-a-concept-reference-term-type)
+4.  [Delete a concept reference term type](#delete-a-concept-reference-term-type)
```
Comment (bmamlin) [inline Concepts/concept_reference_term_type.md]: Remove the term "type(s)"

File: Concepts/concept_reference_term_type.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# Concepts Reference Term Type
+
+## Overview
+
+* A concept term defines a medical coding term or OpenMRS concept dictionary term that could be mapped to a concept. 
+A medical coding term is used to represent a name from a standard medical code (ie.  SNOMED, ICD10, LOINC, etc). OpenMRS 
+concept dictionary terms are used to refer to concepts by concept_id or name of a concept in the concept dictionary (ie. CIEL, PIH, AMPATH, etc.).
```
Comment (bmamlin) [inline Concepts/concept_reference_term_type.md]:
```
* Concept reference terms represent terms within external vocabularies (typically standardized 
terminologies like SNOMED, ICD, LOINC, etc.). OpenMRS concept can be mapped to these 
reference terms through [concept mappings](concept_mapping.md).
```

File: Concepts/concept_reference_term_type.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+* #### List all concepts reference term types.
+
+    Quickly filter concept reference term types with given query parameters.Returns a `404 Not Found` status if concept 
+    reference term type not exists. If user not logged in to perform this action,a `401 Unauthorized` status returned.
+
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of oncepts reference term type object
```
Comment (bmamlin) [inline Concepts/concept_reference_term_type.md]: Does this work for you? In my testing, I was unable to find any concept reference terms using the `q` parameter.

**Review by bmamlin — CHANGES_REQUESTED** [review]

**Comment by Akayeshmantha** [comment]
@bmamlin did the changes.

**Pull request merged**